python_modules/dagster/dagster/_core/storage/db_io_manager.py

```diff
@@ -121,6 +122,13 @@ def __init__(
             self._default_load_type = default_load_type
 
     def handle_output(self, context: OutputContext, obj: object) -> None:
+        # Nones cannot be stored by DB I/O managers. If the output type is set to None/Nothing,
```

Comment:
update comment to be more specific:
"If the output type is set to Nothing, handle_output will not be called. We still need to raise an error when the return value is None, but the typing type is not Nothing